cluster/saltbase/README.md

```diff
@@ -0,0 +1,12 @@
+# SaltStack configuration
+
+This is the root of the SaltStack configuration for Kubernetes. A high
+level overview for the Kubernetes SaltStack configuration can be found [in the docs tree.](../../docs/salt.md)
+
+This SaltStack configuration currently applies to default
+configurations for Debian-on-GCE, Fedora-on-Vagrant, Ubuntu-on-AWS and
+Ubuntu-on-Azure. (That doesn't mean it can't be made to apply to an
+arbitrary configuration, but those are only in-tree supported OS/IaaS
+combinations supported today.) As you peruse the configuration, these
```

Comment:
I see what you mean. I made the above comment before knowing much about salt. Now that I've read some amount of salt documentation, I see that `salt/`, `pillar/`, and `reactor/` are the three main directory trees of salt, so perusing into those specifics is obvious if you know what you're looking for. If you don't know about salt, they seem like completely arbitrary names, and one is immediately motivated to move a directory up and not keep looking.

I guess it depends who you're writing this documentation for. If it's people that know about salt, you needn't say anything more. If it's for salt newbies, some motivation for looking further is needed.

Anyway, I think the links you've added give a nice balance, so this looks good. Just wanted to provide a bit of context that may seem obvious for someone who is knee deep in salt.